.github/workflows/tests.yml

```diff
@@ -25,7 +25,8 @@ jobs:
     - name: Install package
       run: |
         python3 -m pip install pip setuptools wheel
-        python3 -m pip install --ignore-installed -U -q --no-cache-dir -e .[complete]
+        python3 -m pip install pytest
+        python3 -m pip install --ignore-installed -U -q --no-cache-dir -e .
```

Comment:
You might want to encode this into a (conda) environment file, like most dask projects do

Reply:
Next thing on my list, wanted to make it green first